moped-editor/src/views/projects/projectView/ProjectTeam/TeamAutocompleteComponent.js

```diff
@@ -65,10 +70,7 @@ const TeamAutocompleteComponent = ({
   };
 
   return (
-    <FormControl
-      variant="standard"
-      sx={{ width: "100%", mx: theme.spacing(1) }}
-    >
+    <FormControl variant="standard" sx={{ width: "100%", mx: 1 }}>
```

Comment:
Would it make sense to add a css class or `const` to store these styles?

Reply:
idk about right now. i'm hoping that https://github.com/cityofaustin/atd-data-tech/issues/20171 will result in us combining all these different autocompletes into one reusable component that contains the styles in one place.